### DISCUSSION TITLE
Website is down

### DISCUSSION THREAD

**Discussion by simmslabs** [pockethost/pockethost#142] [Q&A]
https://pockethost.io is down

**Comment by benallfree** [comment]
@simmslabs Thanks for the report. Back up again. I did an ill-planned v0.5.6 release today and it has been causing issues. Apologies for the downtime.

**Comment by procrates** [comment]
@benallfree Down again unfortunatly:(  Great system you built!

**Reply by benallfree** [reply]
...aaand back up! 🗡️ 

Sorry for the downtime, this latest round of updates has surfaced some instability. We're getting it fixed one issue at a time, as we see them surface.
